addons/resource.language.en_gb/resources/strings.po

```diff
@@ -4065,7 +4065,9 @@ msgctxt "#1208"
 msgid "Mount SMB shares"
 msgstr ""
 
-#empty string with id 1209
+msgctxt "#1209"
```

Comment:
Could you please provide contextual information for the new strings (here and elsewhere). This makes live of translators a lot easier. Thanks.
#. ...
#: ...